smart-contract-example/README.md

```diff
@@ -0,0 +1,90 @@
+## Smart Contract Workflow Example
+
+This example demonstrates how Kurtosis can be used for smart contract dapp development 
+by using the [`eth-network-package`](https://github.com/kurtosis-tech/eth-network-package) as a local Ethereum network. 
+The `eth-network package` can be used as a low overhead, configurable, and composable alternative to services like
+`hardhat-network`, `ganache` and `anvil`.
```

Comment:
```suggestion
`hardhat-network`, `ganache` and `anvil`. Kurtosis provides the developer more control and flexibility over the testnet they are using - a large reason why the [Ethereum Foundation used Kurtosis to test the Merge](https://www.kurtosis.com/blog/testing-the-ethereum-merge), and why they continue to use Kurtosis for various tests for upcoming upgrades to the network.

While this example covers only the Ethereum blockchain, Kurtosis can be used to locally configure and instantiate other blockchains (e.g. [NEAR](https://docs.near.org/develop/testing/kurtosis-localnet), [Avalanche](https://medium.com/avalancheavax/introducing-kurtosis-a-complete-testing-platform-to-accelerate-development-on-avalanche-6ad7e1147791) and allows you to connect it to any containerized service you wish.
```

Reply:
love this addition! helps communicate better why a dapp developer might choose us over other options.